modules/administration/pages/image-management.adoc

```diff
@@ -299,6 +299,14 @@ RUN /root/add_packages.sh
 RUN rm -f /etc/zypp/repos.d/susemanager:dockerbuild.repo
 ----
 
+//TODO: Replace the "custom-system-info" link
+.Using Custom Info Key-value Pairs as Docker Buildargs
+You can assign custom info key-value pairs to attach information to the image profiles.
+Additionally, these key-value pairs are passed to the Docker build command as `buildargs`.
+
+For more information about the available custom info keys and creating additional ones, see '/refernce/system/custom-system-info.html'
```

Comment:
What should be the actual link text here?

Reply:
For the moment, the link should be "xref:reference:systems/custom-system-info.adoc[]", IMO.